syndiffix/microdata.py

```diff
@@ -191,12 +189,12 @@ def apply_convertors(convertors: list[DataConvertor], raw_data: pd.DataFrame) ->
 
 
 def generate_microdata(
-    buckets: Buckets, convertors: list[DataConvertor], null_mappings: list[float]
+    buckets: Buckets, convertors: list[DataConvertor], null_mappings: list[float], rng: Random
 ) -> list[MicrodataRow]:
```

Comment:
We could create a RNG instance right here and drop the function parameter.

Reply:
Good idea!

Reply:
Though I'm not sure whether it's better to make it stick to the forest for better pseudo randomness...

Reply:
We do this every time we have to produce a microtable. Always using the same sequence might create a bias in the output. I made a small tweak to detach stitching from microdata, otherwise I think it's better to keep it bound to the forest.